server/database/dummy-data/index.js

```diff
@@ -1,6 +1,7 @@
 import dotenv from 'dotenv';
 import createUsers from './users';
 import createOrganisations from './organisations';
+import createLandingPage from '../dummy-data-prod/landingPageContent';
```

Comment:
I think we've agreed to make all files/folders should be lowercase with dashes/hyphens 
I know it's not the best, but at least we should make things consistent across the app
```suggestion
import createLandingPage from '../dummy-data-prod/landing-page-content';
```
also I think you forgot to createSteps here, unless it was intentional

Reply:
done